blog/2024-08-29-updated-roadmap.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Updated Roadmap
+description: Details about the Wing toolchain's recently updated roadmap
+authors:
+  - rybickic
+tags:
+  - cloud-oriented
+  - programming
+  - winglang
+  - roadmap
+hide_table_of_contents: true
+---
+
+Hello Wingnuts!
+
+We recently updated the project's roadmap to share more details about our vision for the project. As the Wing language and toolchain has an ambitious goal, we're hoping this gives you a better idea of what to expect in the coming months.
+
+We want to stabilize as many of the items below as possible by the end of 2025, but we're eagerly interested in you feedback and collaboration, either [through GitHub](https://github.com/winglang/wing) or our [Discord server](https://t.winglang.io/discord).
```

Comment:
Where is 2025 coming from?

Reply:
Discussed on our offsite. Do we have an expected or goal date for 1.0 here? End of 2025 was mentioned but nothing concrete, what are your thoughts? Should we remove the date here? 

Personally I think it would be nice for folks to see where our goal is with dates in mind, but maybe thats just me?

Reply:
I guess the potential risk here is when we mention dates (especially in writing) it can give people the wrong impression that we're making a promise or we're committing to a specific timeline, which I don't think is our intention here.

Reply:
Yeah let's just remove the date, we can always add it if we want!

Reply:
Date removed (for now)